blue_controller_manager/package.xml

```diff
@@ -1,4 +1,4 @@
-<package format="2">
+<package format="3">
```

Comment:
should we update all the package formats to 3 for consistency?

Reply:
done!